text/0630-new-browser-support-policy.md

```diff
@@ -0,0 +1,67 @@
+- Start Date: 2020-05-21
+- Relevant Team(s): All
+- RFC PR: https://github.com/emberjs/rfcs/pull/630
+- Tracking: (leave this empty)
+
+# New Browser Support Policy
+
+## Summary
+
+> In dropping support for IE11 at some point in the future, we will need a new
```

Comment:
so you're saying that we are looking to drop IE11 `at some point in the future`, and this is the "right now" browser support policy. So doesn't that mean that this proposed policy should **include** IE11 and when we drop support for it we remove IE11 from the "official policy"? 🤔

Reply:
I said "This is the proposed policy". How can I make this more clear?